src/sentry/static/sentry/app/views/releases/detail/index.jsx

```diff
@@ -102,33 +136,6 @@ class OrganizationReleaseDetails extends AsyncView {
       </PageContent>
     );
   }
-
-  renderError(error, disableLog = false, disableReport = false) {
-    const has404Errors = Object.values(this.state.errors).find(
-      resp => resp && resp.status === 404
-    );
-    if (has404Errors) {
-      // This catches a 404 coming from the release endpoint and displays a custom error message.
-      const {projects: all_projects} = this.props;
-      const {projects: selected_projects} = this.props.selection;
-
-      return (
-        <PageContent>
-          <Alert type="error" icon="icon-circle-exclamation">
-            {t(
-              'This release may not be in your selected project' +
-                (selected_projects.length > 1 ? 's' : '')
-            )}
-            :{' '}
-            {selected_projects
-              .map(p => all_projects.find(pp => parseInt(pp.id, 10) === p).name)
-              .join(', ')}
```

Comment:
Refactored this a bit since `allProjects` now can be empty and not guaranteed to exist. Do not show the `:` and anything after that unless we find `selected` in `allProjects`.